#!/usr/bin/python3
def raise_exception_msg(message=""):
    """rasie a name exception with message

    Args:
        message: a string

    Return: void
    """
    raise NameError(message)
